### PR TITLE
add docker image stuff

### PR DESCRIPTION
**Pull request by hansvancalster**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          fetch-depth: 0  # OR "2" -> To retrieve the preceding commit.
+
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            dockerfile
+            docker-compose.test.yml
```
Comment (ElsLommelen) [inline .github/workflows/docker.yml:22]: Why should a new docker be build after changes in `docker-compose.test.yml`?  This is just a local test to test in this repo if a docker image can be build, so I see no need to create a tag to build a new docker in this case.

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            test_docker.sh
```
Comment (ElsLommelen) [inline .github/workflows/docker.yml:23]: Same remark for `test_docker.sh`

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            docker
+            renv.lock
+            .github/workflows/docker.yml
```
Comment (ElsLommelen) [inline .github/workflows/docker.yml:26]: And a similar remark for this GHA: why should a new docker be build after changes in this GHA (that even doesn't use a docker)?

**Review by ElsLommelen — COMMENTED** [review]
Here already some small comments (which are similar to my remark in the googledoc, so maybe you were to change it anyway?)

File: .github/workflows/docker.yml (hunk 1 above)
Comment (hansvancalster) [reply .github/workflows/docker.yml]: You're right. I fixed it.

**Pull request merged**